src/screens/Onboarding/StepProfile/AvatarCircle.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import {
+  LayoutAnimation,
+  Pressable,
+  PressableProps,
+  StyleSheet,
+  View,
+} from 'react-native'
+import {atoms as a, useTheme} from '#/alf'
+import {usePhotoLibraryPermission} from 'lib/hooks/usePermissions'
+import {openPicker} from 'lib/media/picker.shared'
+import {isNative, isWeb} from 'platform/detection'
+import {openCropper} from 'lib/media/picker'
+import {compressIfNeeded} from 'lib/media/manip'
+import {Image} from 'expo-image'
+import {TimesLarge_Stroke2_Corner0_Rounded as Times} from '#/components/icons/Times'
+import {Camera_Stroke2_Corner0_Rounded as Camera} from '#/components/icons/Camera'
+import {useAvatar, useSetAvatar} from '#/screens/Onboarding/StepProfile/index'
+
+export function AvatarBottomButton({...props}: PressableProps) {
+  const t = useTheme()
+
+  return (
+    <Pressable
+      {...props}
+      style={[
+        a.absolute,
+        a.rounded_full,
+        a.align_center,
+        a.justify_center,
+        t.name === 'light' ? t.atoms.bg_contrast_800 : t.atoms.bg_contrast_200,
```

Comment:
A thought about this. Maybe we can allow passing a param to `useTheme()` to get the dark or light theme? Suppose we could import the palette directly, but the consistency with just being able to still use the `useTheme()` hook feels better.